immunopepper/utils.py

```diff
@@ -329,20 +326,23 @@ def get_idx(countinfo, sample, gene_idx):
     return Idx(gene_idx, sample_idx)
 
 
-def create_libsize(expr_distr_fp, output_fp, sample, debug=False):
+def create_libsize(expr_distr_fp, libsize_fp, out_dir, mutation_mode, parallel=1):
```

Comment:
It does matter because the library size is calculated over the expression of genes. And the gene expression files are stored together with the result of each gene batch. These gene batches are in several batch folders, which names do contain the mutation mode. Batches folder are not shared across mutation modes because they contain SUCCESS_ files used to check whether one run of a given mutation mode run as finished computing the batch.

Reply:
It is true that while the kmer files, the peptides files do depend of the mutation mode, the gene expression does not. So if we are running 4 different runs with mutation mode ref, germline, somatic_and_germline, somatic, we are computing and saving 4 times the gene expression.

Reply:
Changing this require quite some change in the folder output structure, and right now this is not ideal for my experiments. But I can keep it in mind and update it at a later time point.

Reply:
ok, please add to the TODO list. Can be resolved later, but should be resolved eventually.